workbench-core/base/src/aws/helpers/dynamoDB/dynamoDBService.ts

```diff
@@ -271,6 +273,24 @@ export default class DynamoDBService {
     return query;
   }
 
+  /**
+   * Places a DDB Update call and formats the response.
+   *
+   * @param updateParams - {@link UpdateParams} object to pass to the update request
+   * @returns a {@link UpdateUnmarshalledOutput} item where Attributes is unmarshalled
+   */
+  public async updateExecuteAndFormat(updateParams: UpdateParams): Promise<UpdateUnmarshalledOutput> {
```

Comment:
This name is kind of a mouthfull. What about something like 'updateItems'? I'm not sure we want the function names or types to have details about the inner implementation. 

Same thing goes for the return object. Maybe 'UpdateItemsResponse'?

Reply:
This is only a single update item call so plural wouldn't work. I wanted to make it easy to differentiate between this function and normal update. Looking at `.update()` and `.updateItem()`, I would not know the difference or which one to use.

Open to other suggestions that still easily paint the difference.

Reply:
If you wanted to rename:

Memory is a little hazy, are we looking to deprecate the old `.update().execute()` method in favor of this method? If so, it might make sense to mark the old method as deprecated in TSDoc and point the user to use this method instead. Then the naming might not be as confusing.